src/app/core/session/session-service/synced-session.service.spec.ts

```diff
@@ -171,7 +171,7 @@ describe("SyncedSessionService", () => {
     // Initially the user is logged in
     expectAsync(result).toBeResolvedTo(LoginState.LOGGED_IN);
     // After remote session fails the user is logged out again
-    expect(sessionService.loginState.value).toBe(LoginState.LOGIN_FAILED);
+    expect(sessionService.loginState.value).toBe(LoginState.LOGGED_OUT);
```

Comment:
why should this not be expected to be state "failed"?

Reply:
The failed state does not really make sense in this context anymore as we are reloading the app on logout (due to various services that only work once per session and the different databases per users) therefore the session state here is not relevant and we are now just using the default logout state.